mitodl/mit-open
# add SignupPopover to learning resource drawer bookmark button

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/5505

### Description (What does it do?)
This PR adds some functionality that was missed in https://github.com/mitodl/mit-open/pull/1537. If you are logged out, the bookmark button should still show in the learning resource drawer. When clicking this button, the same "signup popover" that shows when you click the bookmark button on the card show show.

### Screenshots (if appropriate):
![image](https://github.com/user-attachments/assets/e43aca2f-e858-4d20-a5f4-d6f0c86f6c57)
![image](https://github.com/user-attachments/assets/17292535-a93c-47dd-95b5-dfbfb2f7f1ca)

### How can this be tested?
 - Spin up this branch of `mit-learn`
 - Make sure you have at least a few learning resources in your database
 - Open an incognito window or log out
 - Visit the search page
 - Click on any result to open the learning resource drawer
 - Within the drawer, click the bookmark button
 - You should see a popover as seen in the screenshots above
 - Click "Sign Up"
 - Log in or create a new account
 - Verify that you are returned to the page with the learning resource drawer open, and you can now click the bookmark button and add the resource to a list
